casacore/fitting/fitting.py

```diff
@@ -100,13 +99,13 @@ def init(self, n=0, ftype="real", colfac=1.0e-8, lmfac=1.0e-3, fid=0):
             return False
 
     def _gettype(self, ftype):
-        if isinstance(ftype, str):
+        if isinstance(ftype, string_types):
             ftype = ftype.lower()
             if ftype not in self._typeids:
                 raise TypeError("Illegal fitting type")
             else:
                 return self._typeids[ftype]
-        elif isinstance(ftype, int):
+        elif isinstance(ftype, six.integer_types):
```

Comment:
Should `integer_types` be imported at the top, for consistency with other modules?